libs/langchain/langchain/chains/base.py

```diff
@@ -147,11 +147,12 @@ def invoke(
             self.metadata,
         )
         new_arg_supported = inspect.signature(self._call).parameters.get("run_manager")
-
         run_manager = callback_manager.on_chain_start(
             None,
             inputs,
             run_id,
+            class_name=self.name,
```

Comment:
This is a big decision! 

`on_chain_start` here is an event dispatcher. By updating the information sent here the code sets the shape of dispatched on chain start events.

i.e., we should probably not be including `class_name` and `lc_id` in here, instead everything can be placed in the `name` logic if any additional attributes are needed

Reply:
I'm going to remove these additions since they're not necessary or desirable